webapp/store/index.js

```diff
@@ -16,6 +16,24 @@ import {
 } from '../plugins/plugins';
 import * as reducers from './reducers';
 
+function errorCatcherMiddleware(errorHandler) {
+  return function(store) {
+    return function(next) {
+      return function(action) {
+        try {
+          return next(action);
+        } catch (err) {
+          const message = `There was an error in the middleware for the action: ${
```

Comment:
Should these types of messages live in a `messages.js` type of file?

Reply:
Do you think it makes sense to put such specific messages in a separate file? I could see it being useful if a message is going to be used in multiple locations but this is specific to middleware.

Reply:
I think that it could be, even though the message is only used in one place. See https://github.com/bcoin-org/bcrypto/blob/master/vendor/secp256k1/src/messages.h

You could make the argument that JS vs C have different styles and that the message needs string interpolation, but you could make the message a function that does the interpolation for you. Maybe that adds complexity, I'm not super opinionated but lean more towards keeping them all in one file

Reply:
Yeah, I feel like this is more of a C convention than a JS one. It also makes it more cluttered since there is no sense of header files in JS and so looking at the directory structure you wouldn't be able to tell that that's all that is going on there (i.e. you can tell what's what in a C project by looking at the extensions). You could make these JSON I suppose, but even looking at JJ's own code he's not using an equivalent messages file for his JS that I've seen.